### PR TITLE
feat(components/google-cloud): Add API documentation

### PR DESCRIPTION
**Pull request by SinaChavoshi**
Adding API documentation using Sphinx

### CODE REVIEW

File: components/google-cloud/docs/tests.aiplatform.rst [hunk 1]
```diff
@@ -0,0 +1,37 @@
+tests.aiplatform package
```
Comment (sasha-gitg) [inline components/google-cloud/docs/tests.aiplatform.rst:1]: I'm not sure we should include documentation for the tests.

File: components/google-cloud/docs/source/index.rst [hunk 2]
```diff
@@ -0,0 +1,20 @@
+.. google_cloud_pipeline_components documentation master file, created by
+   sphinx-quickstart on Fri May  7 04:23:22 2021.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to google_cloud_pipeline_components's documentation!
+============================================================
```
Comment (sasha-gitg) [inline components/google-cloud/docs/source/index.rst:7]: We may want to reference the python-aiplatform repo and docs here:

- https://github.com/googleapis/python-aiplatform
- https://googleapis.dev/python/aiplatform/latest/aiplatform.html#

**Review by sasha-gitg — APPROVED** [review]
/lgtm
/approve

File: components/google-cloud/docs/tests.aiplatform.rst (hunk 1 above)
Comment (SinaChavoshi) [reply components/google-cloud/docs/tests.aiplatform.rst:1]: correcting in a follow up PR.

File: components/google-cloud/docs/source/index.rst (hunk 2 above)
Comment (SinaChavoshi) [reply components/google-cloud/docs/source/index.rst:7]: Also correcting in follow up PR.

File: components/google-cloud/docs/tests.aiplatform.rst (hunk 1 above)
Comment (SinaChavoshi) [reply components/google-cloud/docs/tests.aiplatform.rst:1]: addressed in #5611

File: components/google-cloud/docs/source/index.rst (hunk 2 above)
Comment (SinaChavoshi) [reply components/google-cloud/docs/source/index.rst:7]: Addressed  in #5611

**Pull request merged**